modules/system/defaults/dock.nix

```diff
@@ -163,6 +163,24 @@ in {
       '';
     };
 
+    system.defaults.dock.magnification = mkOption {
+      type = types.nullOr types.bool;
+      default = false;
```

Comment:
Should we make `null` the default so we don't override this setting for users who don't set a value in `nix-darwin`?

Reply:
sure, you're right! I have prepared these changes sometime ago, and should have pruned the code before creating PR 🙂